.circleci/config.yml

```diff
@@ -27,7 +22,9 @@ jobs:
       - run:
           name: install dependencies
           command: |
-            bundle install --jobs=4 --retry=3 --path vendor/bundle
+            bundle config set path 'vendor/bundle'
+            bundle config set clean 'true'
```

Comment:
Will this option help with something?

Reply:
if restore_cache with partly-matched key, cleaning unused gems is good for slimmer cache, but no restoration, so no meaning...